### ISSUE TITLE
A contributor can press a "See all annotations" button or link on an artifact to go to a view of annotations filtered by the current artifact

### ISSUE THREAD

**Issue by mlncn** [PECE-project/drupal-pece#437]
- add see all annotations of this artifact button, linking to a view of annotations filtered to that artifact.

**Comment by mlncn** [comment]
@lindsaypoirier @mikefortun i updated the button to say "See all Annotations of this Artifact", but can anything other than an artifact have annotations?

It would be easy to change this to say the specific type of artifact (already done in the tooltip) and pretty trivial to have ti say 'Artifact' if it is an artifact and … something else if it is not.

**Comment by lindsaypoirier** [comment]
Yes essays have artifacts. I think it would be fine to say See Annotations

*Lindsay Poirier*
she/her/hers
Assistant Professor
Statistics and Data Sciences
Smith College
Personal website <https://lindsaypoirier.github.io/>

*Recent Work*


   - Poirier, Lindsay, Derrick Salvatore, Phil Brown, Alissa Cordner, Kira
   Mok, and Nicholas Shapiro. 2024. “Per- and Polyfluoroalkyl Substance
   Exposure Risks in US Carceral Facilities, 2022.” *American Journal of
   Public Health*, March, e1–10. https://doi.org/10.2105/AJPH.2024.307571.
   - Poirier, Lindsay, Dexter Antonio, Makenna Dettmann, Tiffany Eng,
   Jennifer Ganata, Sujoy Ghosh, Mirthala Lopez, Ranesh Karma, Asiya Natekal,
   and Catherine Brinkley. 2024. “Making Plans Findable, Accessible,
   Interoperable, and Reusable with Data Infrastructure: A Search Engine for
   Constructing, Analyzing, and Visualizing Planning Documents.” *Environment
   and Planning B: Urban Analytics and City Science*, January.
   https://doi.org/10.1177/23998083241227471.
   - Poirier, Lindsay. "Attending to the Cultures of Data Science Work." *Data
   Science Journal* 22, no. 6 (2023): 1-7.
   https://doi.org/10.5334/dsj-2023-006
